src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -248,8 +243,8 @@ private FedoraResource getBaseVersion(final FedoraResource resource) {
 
         // Transform the effectiveRoles from a Set to a List.
         return effectiveRoles.entrySet().stream()
-            .map(x -> new AbstractMap.SimpleEntry<>(x.getKey(), new ArrayList<>(x.getValue())))
-            .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+            .map(roleSet -> new AbstractMap.SimpleEntry<>(roleSet.getKey(), new ArrayList<>(roleSet.getValue())))
```

Comment:
Okay, otherwise, much as I like the Stream API, I think `static import` Guava's `Maps.transformValues` is actually much shorter and clearer here: `return transformValues(effectiveRoles, ArrayList::new)`.